src/PendingCalls/TestCall.php

```diff
@@ -172,15 +172,28 @@ public function skip(Closure|bool|string $conditionOrMessage = true, string $mes
         return $this;
     }
 
+    /**
+     * Skips the current test if the given test is running on given os family.
+     */
+    public function skipOsFamily(string ...$osFamilies): self
```

Comment:
What value do I need to pass for MacOS?

Reply:
I think it's `Darwin`

Reply:
https://github.com/php/php-src/blob/fced34ee1dd1af2439e88fecda299de44da4b860/main/php.h#L40-L53